sdk/keyvault/azure-keyvault-certificates/samples/backup_restore_operations_async.py

```diff
@@ -48,10 +48,10 @@
         # Let's create a certificate for your key vault.
         # if the certificate already exists in the Key Vault, then a new version of the certificate is created.
         # An async poller is returned.
-        create_certificate_poller = await client.create_certificate(name=cert_name)
+        create_certificate_poller = client.create_certificate(name=cert_name)
 
-        # Awaiting the poller will return a certificate if creation is successful, and will return the failed
-        # CertificateOperation if not.
+        # Awaiting the poller will return a certificate if creation is successful,
+        # and will return the failed CertificateOperation if not.
```

Comment:
I would get the result of the poller and do something with it, I believe it will make the sample more valuable

Reply:
doing that now, thanks!